contracts/CO30_Matias_SymphonyOfDestruction.tem_vinjesvingenc/initPlayerLocal.sqf

```diff
@@ -0,0 +1,10 @@
+#include "do_not_edit\script_component.hpp"
+#include "do_not_edit\initPlayerLocal.sqf"
+
+
+[
+  {time > 1},
+  {
+    execVM "scripts\detector.sqf";
+  }
+] call CBA_fnc_waitUntilAndExecute;
```

Comment:
no execvm
```suggestion

mission_fnc_detector = compile preprocessFileLineNumbers "scripts\detector.sqf";

[
  {time > 1},
  {
    call mission_fnc_detector;
  }
] call CBA_fnc_waitUntilAndExecute;
```

Reply:
Emerson did it that way during his Probing contract. His code had a comment that said that the execVM only executed at the beginning of the mission due to how it's written in his code.

Reply:
Ok